apps/src/sites/studio/pages/programming_expressions/show.js

```diff
@@ -33,6 +30,9 @@ $(document).ready(() => {
 
   const categoriesForNavigation = getScriptData('categoriesForNavigation');
   const currentCategoryKey = getScriptData('currentCategoryKey');
+
+  prepareBlocklyForEmbedding(customBlocksConfig, programmingEnvironmentName);
```

Comment:
This was moved down just so that we could do it after we had the programming name.